src/Security/Firewall/ExceptionListener.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AppBundle\Security\Firewall;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Http\Firewall\ExceptionListener as BaseExceptionListener;
+
+class ExceptionListener extends BaseExceptionListener
+{
+    protected function setTargetPath(Request $request)
+    {
+        if (
+            $request->isXmlHttpRequest()
+            || mb_strpos($request->getPathInfo(), '/api/') === 0
```

Comment:
What about using the parameter `%env(API_PATH_PREFIX)%`?
In this case if we change its value, we will have no need to modify this line.

Reply:
YES! thanks